packages/react-test-renderer/shallow.js

```diff
@@ -7,10 +7,4 @@
  * @flow
  */
 
-'use strict';
-
-const ReactShallowRenderer = require('./src/ReactShallowRenderer');
-
-// TODO: decide on the top-level export form.
-// This is hacky but makes it work with both Rollup and Jest.
-module.exports = ReactShallowRenderer.default || ReactShallowRenderer;
+export {default} from './src/ReactShallowRenderer';
```

Comment:
Isn't this gonna break `const ShallowRenderer = require('react-test-renderer/shallow')`? Is this not a breaking change?

Reply:
No, Rollup makes this into `module.exports = `. (Babel doesn't which is what Jest is struggling with.)

Reply:
🤯 I had no idea